Gemfile.lock

```diff
@@ -1,125 +1,125 @@
 PATH
   remote: .
   specs:
-    cm_page_builder-rails (0.2.1)
-      rails (~> 6.0.0)
+    cm_page_builder-rails (0.2.2)
```

Comment:
@manikandan0603 @mikevic @anbublacky @DeboDevelop - I feel we need to change the structure of the gem and remove the gemfile.lock. We are loading a lot of gems multiple times first using the page builder and then on the project which seems redundant.

Reply:
@AdityaTiwari2102 - Can you send a PR for this? I'd be curious to see how this would look.